polaris-react/src/components/TopBar/TopBar.stories.tsx

```diff
@@ -49,15 +55,42 @@ export function Default() {
     <TopBar.UserMenu
       actions={[
         {
-          items: [{content: 'Back to Shopify', icon: ArrowLeftMinor}],
+          items: [
+            {
+              active: true,
+              content: 'Jaded Pixel',
+              prefix: <Avatar size="small" name="Jaded Pixel" />,
+              suffix: <Icon source={TickSmallMinor} />,
+              truncate: true,
+            },
+            {
+              content: 'Jaded Pixel 2.0',
+              prefix: <Avatar size="small" name="Jaded Pixel 2.0" />,
+              truncate: true,
+            },
+            {
+              content: 'View all 3 stores',
+              prefix: <Icon source={SearchMinor} />,
+            },
+          ],
         },
         {
           items: [{content: 'Community forums'}],
         },
+        {
+          items: [{content: 'Help Center'}],
+        },
+        {
+          items: [{content: 'Keyboard shortcuts'}],
+        },
+        {
+          title: 'Dharma Johnson',
+          items: [{content: 'Manage account'}, {content: 'Log out'}],
+        },
```

Comment:
Are all user menus going to have the store switcher eventually instead of the Stores action list item that links out to an identity page? If not, should this use case be documented as its own story since it's unique to merchants who have the store switcher?

Reply:
Ah okay! In that case, should we:
-  Make this a layout change internal to the component instead of adding a `customActivator` prop, since the pattern is being updated across the board?
- Add an optional `logo` prop that renders a `Thumbnail` to the right of the `name` (instead of an `avatar` on the left) when provided, and make `initials` optional?